### PR TITLE
Revheads are now stealthier/harder to metagame

### PR DESCRIPTION
**Pull request by Robustin**
:cl: Robustin
tweak: Revheads no longer spawn with chameleon glasses or a spraycan, instead they will start with a cybernetic security HUD implanted into their eyes. 
/:cl:

Instantly confirming revheads based on their eyewear and spraycan possession is silly. I have personally been metagamed as an assistant for wearing sunglasses and I will admit that if I'm hunting for revheads I will absolutely prioritize anyone wearing eyewear and I will often check that eyewear before deciding to implant them.

According to stats heads win more than revs so this seems like an uncontroversial change - so naturally I'm expecting a huge shitshow in the comments.

### CODE REVIEW

File: code/game/gamemodes/revolution/revolution.dm [hunk 1]
```diff
@@ -154,22 +154,16 @@
...
 	)
 	var/where = mob.equip_in_one_of_slots(T, slots)
-	var/where2 = mob.equip_in_one_of_slots(C, slots)
-	mob.equip_in_one_of_slots(R,slots)
-
-	if (!where2)
-		to_chat(mob, "The Syndicate were unfortunately unable to get you a chameleon security HUD.")
-	else
-		to_chat(mob, "The chameleon security HUD in your [where2] will help you keep track of who is mindshield-implanted, and unable to be recruited.")
+	S.Insert(mob, special = 0, drop_if_replaced = FALSE)
```
Comment (Cyberboss) [inline code/game/gamemodes/revolution/revolution.dm]: special = FALSE

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/revolution/revolution.dm (hunk 1 above)
Comment (Robustin) [reply code/game/gamemodes/revolution/revolution.dm]: I dont even know what special DOEEESSSS

**Comment by kevinz000** [comment]
Unzips health analyzer

**Comment by Repukan** [comment]
Revheads or gangheads?

**Review by Cyberboss — APPROVED** [review]

**Comment by praisenarsie** [comment]
@Repukan gang doesn't exist anymore 💢

**Comment by DoctoreMad** [comment]
In order to expose the amount of coders that do not play this game, I will post this single comment and wait to see how many people remember to do it in the next rounds:
You can smash (non-head) revs in the head with blunt weapons in order to make them remember their allegiance.

As for actual commentary on the PR: ~Speed merge please~ Why the fuck do revheads have spray cans? Facespray stun?

**Comment by jughu** [comment]
@Robustin SOMETHING SOMETHING MED HUDS DETECTING IMPLANTS BY EXAMINING THEM INB4 FIND REVHEAD ROUNDSTART.

**Comment by AnonymousNow** [comment]
No, actually, that works out fine for me.

CMOs have long been the weakest heads in revolutions, constantly passed up for survivability by people imagining that they'd have plenty of time to cook up medicines and toxins in chemistry during a revolution, never seeing the reality of them being swarmed in the opening seconds.

This is a factor that helps to give the CMO the hint of a semblance of a fighting chance.

**Comment by Repukan** [comment]
You're going to still get a dick in your mouth when they swarm you lmao

**Comment by CPTANT** [comment]
Uhm you can just simply not wear the chameleon glasses you know?

**Comment by Robustin** [comment]
Then you get arrested for flashing and they find em on you anyway. 

I just don't like the idea of confirming revheads based on spraycans and eyewear. I wasn't aware implants showed up on medscanners but I might try to find a way around that if it ends up getting meta'd.

**Comment by XDTM** [comment]
I thought this was gonna be something about reacting to implants like cultists; i still think it might be cool if implanting wasn't the ultimate headrev test.

Then again, sec already mass murders enough without making even implanted crew worthy of suspicion.

**Comment by Robustin** [comment]
There's absolutely no way we could allow for implanted headrevs to exist.

**Comment by ExcessiveUseOfCobblestone** [comment]
what you need to do is make the chameleon huds simply not function for non-Rheads instead of moving the powergame goalpost.

You're not actually fixing the problem, you're just making it require a health analyzer.

**Comment by lordpidey** [comment]
@Robustin, you can brainswap a headrev into an implanted body.  It's hilarious.

**Comment by Cyberboss** [comment]
I agree with @ExcessiveUseOfCobblestone

**Comment by Bawhoppen** [comment]
Rev is due for the chopping block anyways so

**Comment by lordpidey** [comment]
Rev has it's purpose.  Sometimes we just want a round of absolute chaos.

**Comment by BeeSting12** [comment]
@Robustin 
The solutions to not getting caught by cham HUDs are just... don't wear them. It's that simple. An implant would be just as easy to tell by sec since it only requires a health analyzer. This could even make the new meta running around health analyzing everyone till you find the revhead. I definitely support the removal of the absolutely useless spraycan. (Unless it's for use as a ghetto pepperspray.)

**Comment by KorPhaeron** [comment]
This is actually worse for revheads since they cant ditch these

But either way it doesnt matter much because of loyalty implant spam

**Comment by Arianya** [comment]
Its kinda veering off topic but loyalty implant spam is kinda unavoidable unless rev is meant to be "sec kills everyone on station on suspicion of being a traitor"

**Comment by jughu** [comment]
It doesn't matter yeah besides its chameleon even if they meta they be going for you anyways to implant

**Comment by ExcessiveUseOfCobblestone** [comment]
>  I have personally been metagamed as an assistant for wearing sunglasses and I will admit that if I'm hunting for revheads I will absolutely prioritize anyone wearing eyewear and I will often check that eyewear before deciding to implant them.

It does matter!!!!

**Comment by jughu** [comment]
implant doesnt get spend on revhead anyways so who cares

**Comment by Kerbin-Fiber** [comment]
Sometimes they do get implanted, I did once but it is still very rare.

**Comment by jughu** [comment]
REMEMBER WHEN WE DIDNT HANDHOLD AND DIDNT GAVE REVS CHAM SECHUDS SO REVS HAD TO FIGURE WHOS LOYALITY IMPLANTED WITH NO KNOWLEDGE AND FLASH PROTECTION

**Comment by BeeSting12** [comment]
Remove implants and make beating heads the only option to deconvert.

**Comment by Arianya** [comment]
@BeeSting12 "Oops I was trying to deconvert this guy and he died! Oh well, he might have been a rev and its a rev crisis so I'll just keep him in the brig mortuary for now and take him to cloning later."

5 minutes later

"Oops I was trying to deconvert this guy and he died! Oh well..."

Don't incentivize removing people from the round when the system we have does a better job.

**Comment by Supermichael777** [comment]
One of the reasons that I feel rev is so good is that de-conversion is just as fast as conversion. Both our other conversion modes have an agonizingly slow de-conversion process that ends up with half the sec team in the brig playing egg timer. Headbeating is slow and frequently results in death due to lack of medical supplies and the tendency of shittty sec officers to jump on the poor guy with their own lethals.

**Comment by Cyberboss** [comment]
Just stay the fuck away from health scanners

**Comment by CPTANT** [comment]
Ironically making revheads only easier to meta.

**Comment by praisenarsie** [comment]
> grabs health scanner
grabs stunprod

valid time

**Pull request merged**